standard/file/File.yaml

```diff
@@ -0,0 +1,20 @@
+---
+type: template
+parameters:
+  - name: metadata
+    default: null
```

Comment:
`null` or `[]`

Reply:
use graviti.portex to run your code first

Reply:
`null` is ok, it will be treated as empty fields.